now.json

```diff
@@ -27,8 +27,8 @@
       "use": "@now/next"
     },
     {
-      "src": "api/index.js",
-      "use": "@now/node-server"
+      "src": "api/index.tsx",
+      "use": "@dblechoc/node-server"
     },
```

Comment:
custom fork of node-server, while waiting for some code to be published, PR just got merged: https://github.com/zeit/now-builders/pull/128#event-2017191650

Reply:
they're usually really fast about this. Might hold off on merging this PR for a few hours to see if they cut another release. (totally cool if we're on `@now/node-server@canary)

Reply:
yes once a new version is published i'll update the PR

Reply:
Looks like they still haven't cut a new release, maybe if we ask them to they will release a new canary?

Super eager to get this PR merged! 🤩